carla_ground_truth_integration/docker-compose.yml

```diff
@@ -17,7 +17,7 @@ version: '2'
 
 services:
   roscore:
-    image: usdotfhwastol/carma-base:carma-system-4.2.0
+    image: usdotfhwastol/carma-base:carma-system-4.5.0
```

Comment:
ORG and tag here needs to change. all occurrences in this file.

Reply:
No idea why this changeset is showing up like this. On my local machine this is already correct.

Reply:
Did not have the changes saved. Just pushed them